notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -170,7 +170,7 @@
       "source": [
```

Comment:
<p>Combine these 2 text cells</p><p> </p><h2>Define training specification -&gt; Define the training specification</h2><h2><br></h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Done.